examples/sphere/shallow_water.jl

```diff
@@ -416,106 +435,147 @@ if haskey(ENV, "CI_PERF_SKIP_RUN") # for performance analysis
     throw(:exit_profile)
 end
 
-sol = @timev OrdinaryDiffEq.solve!(integrator)
-
-@info "Test case: $(test_name)"
-@info "  with α: $(test_angle_name)"
-@info "Solution L₂ norm at time t = 0: ", norm(Y.h)
-@info "Solution L₂ norm at time t = $(T): ", norm(sol.u[end].h)
-@info "Fluid volume at time t = 0: ", sum(Y.h)
-@info "Fluid volume at time t = $(T): ", sum(sol.u[end].h)
-
-if test_name == steady_state_test_name ||
-   test_name == steady_state_compact_test_name
-    # In these cases, we use the IC as the reference exact solution
-    @info "L₁ error at T = $(T): ", norm(sol.u[end].h .- Y.h, 1)
-    @info "L₂ error at T = $(T): ", norm(sol.u[end].h .- Y.h)
-    @info "L∞ error at T = $(T): ", norm(sol.u[end].h .- Y.h, Inf)
-    # Pointwise final L₂ error
-    Plots.png(
-        Plots.plot(sol.u[end].h .- Y.h),
-        joinpath(path, "final_height_L2_error.png"),
-    )
-    linkfig(
-        relpath(
-            joinpath(path, "final_height_L2_error.png"),
-            joinpath(@__DIR__, "../.."),
-        ),
-        "Absolute error in height",
-    )
-    # Height errors over time
-    relL1err = Array{Float64}(undef, div(T, dt))
-    for t in 1:div(T, dt)
-        relL1err[t] = norm(sol.u[t].h .- Y.h, 1) / norm(Y.h, 1)
+if usempi
+    walltime = @elapsed sol = OrdinaryDiffEq.solve!(integrator)
+    ClimaComms.iamroot(context) && println("walltime = $walltime (sec)")
+else
+    sol = @timev OrdinaryDiffEq.solve!(integrator)
```

Comment:
This is fine, I'm not sure if this makes a functional difference to keep the expressions in lines `439` and `442` identical in the MPI / serial runs ; Just so I understand the use of `@elapsed` here - are there substantial differences in the evaluations in `@elapsed` and `@timev`, e.g. `gc` inclusion?

Reply:
`@timev` is very informative, but generates garbled output for the distributed case. Therefore, we use `@elapsed` for distributed cases, but still use `@timev` for serial case.